docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md

```diff
@@ -349,15 +348,15 @@ Connect your framework to the iOS project manually:
    import shared
    ```
 
-2. To check that it is properly connected, use the `greeting()` function from the shared module of your cross-platform app:
+2. To check that it is properly connected, use the `greet()` function from the shared module of your cross-platform app:
 
    ```swift
    import SwiftUI
    import shared
    
    struct ContentView: View {
        var body: some View {
-           Text(Greeting().greeting())
+           Text(Greeting().greet())
```

Comment:
JFYI

source code of `class Greeting` with `fun greet(): String` is here
https://kotlinlang.org/docs/multiplatform-mobile-create-first-app.html#update-your-application

Reply:
BTW
I just realized that the screen capture in this [page](https://kotlinlang.org/docs/multiplatform-mobile-create-first-app.html#update-your-application) is still the old edition using old function name `greeting()` instead of the new function name `greet()`

so I send a fix for this screen capture: https://github.com/JetBrains/kotlin-web-site/pull/3489/commits/4085922dae80ca895ef749b4bc3761b99a7ca95f